### PR TITLE
Increase actions in Throttle test

### PR DESCRIPTION
**Pull request by cbickel**
PG1#310 is running

### CODE REVIEW

File: tests/src/limits/ThrottleTests.scala [hunk 1]
```diff
@@ -174,23 +175,24 @@ class ThrottleTests
...
                 wsk.action.invoke(name, Map("payload" -> 10.milliseconds.toMillis.toJson), expectedExitCode = DONTCARE_EXIT)
             }
 
-            // Sleep 3 seconds to let the background thread get the newest values
+            // Sleep 5 seconds to let the background thread get the newest values (refreshes every 2 seconds)
             Thread.sleep(3.seconds.toMillis)
 
-            val endResults = untilThrottled(10) { () =>
+            // Start only 119 activations to avoid being blocked by the invokesPerMinute limit
+            val endResults = untilThrottled(18) { () =>
```
Comment (markusthoemmes) [inline tests/src/limits/ThrottleTests.scala]: this equals `maximumInvokesPerMinute - slowInvokes - fastInvokes`?

Comment (cbickel) [inline tests/src/limits/ThrottleTests.scala]: if maximumInvokesPerMinute is greater than the concurrent ones, yes.
I will change it. Good catch.

File: tests/src/limits/ThrottleTests.scala [hunk 2]
```diff
@@ -174,23 +175,25 @@ class ThrottleTests
...
             val fastResults = untilThrottled(fastInvokes.toInt) { () =>
                 wsk.action.invoke(name, Map("payload" -> 10.milliseconds.toMillis.toJson), expectedExitCode = DONTCARE_EXIT)
             }
 
-            // Sleep 3 seconds to let the background thread get the newest values
+            // Sleep 5 seconds to let the background thread get the newest values (refreshes every 2 seconds)
             Thread.sleep(3.seconds.toMillis)
 
-            val endResults = untilThrottled(10) { () =>
+            // Start only 119 activations to avoid being blocked by the invokesPerMinute limit
```
Comment (markusthoemmes) [inline tests/src/limits/ThrottleTests.scala]: `// start 1 invoke less than the maximum per minute to avoid getting rate throttled`

Comment (perryibm) [inline tests/src/limits/ThrottleTests.scala]: Thread.sleep still says 3 seconds even though comments says 5 seconds

File: tests/src/limits/ThrottleTests.scala [hunk 3]
```diff
@@ -174,23 +175,25 @@ class ThrottleTests
...
+                wsk.action.invoke(name, Map("payload" -> 15.seconds.toMillis.toJson), expectedExitCode = DONTCARE_EXIT)
             }
 
             // Create queue length quickly, drain fast
             val fastResults = untilThrottled(fastInvokes.toInt) { () =>
                 wsk.action.invoke(name, Map("payload" -> 10.milliseconds.toMillis.toJson), expectedExitCode = DONTCARE_EXIT)
             }
 
-            // Sleep 3 seconds to let the background thread get the newest values
+            // Sleep 5 seconds to let the background thread get the newest values (refreshes every 2 seconds)
```
Comment (markusthoemmes) [inline tests/src/limits/ThrottleTests.scala]: typo, still 3

**Comment by markusthoemmes** [comment]
rebased, fixed some nits 4993d80b10b18ab6c934471718f3c22f01920459

**Pull request closed without merging**